node/src/test/scala/coop/rchain/node/RhoExprToParSpec.scala

```diff
@@ -0,0 +1,70 @@
+package coop.rchain.node
+
+import cats.syntax.all._
+import coop.rchain.models.syntax._
+import coop.rchain.node.api.WebApi._
+import org.scalacheck.Arbitrary._
+import org.scalacheck.{Arbitrary, Gen}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.propspec.AnyPropSpec
+import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
+
+class RhoExprToParSpec extends AnyPropSpec with ScalaCheckDrivenPropertyChecks with Matchers {
+  property("WebAPI RhoExpr => Par => RhoExpr conversion work correct") {
+    forAll { rhoExpr: RhoExpr =>
+      rhoExprToRhoExpr(rhoExpr) shouldBe rhoExpr.some
+    }
+  }
+
+  // Converts RhoExpr to Par and then back to RhoExpr
+  private val rhoExprToRhoExpr = rhoExprToParProto _ andThen exprFromParProto
+
+  implicit private def aArb: Arbitrary[RhoExpr] = {
+
+    val numOfElements = 2 // StackOverflowError with n >= 3
+
+    // def genExprPar   = withList(ExprPar)
```

Comment:
Converting to `Par` removes nesting, as noted in [message](<https://discord.com/channels/391435113415573504/410182072054054912/992439986412584980>). Therefore, comparing the original `ExprPar` and the received one leads to a test error. We need to decide if we can ignore nesting in this case.